test/utils/runners.go

```diff
@@ -633,6 +634,10 @@ func (config *RCConfig) applyTo(template *v1.PodTemplateSpec) {
 	if config.MemRequest > 0 {
 		template.Spec.Containers[0].Resources.Requests[v1.ResourceMemory] = *resource.NewQuantity(config.MemRequest, resource.DecimalSI)
 	}
+	if config.GpuRequest > 0 {
+		template.Spec.Containers[0].Resources.Limits["nvidia.com/gpu"] = *resource.NewQuantity(config.GpuRequest, resource.DecimalSI)
+		// TODO[losipiuk] should I use framework.NVIDIAGPUResourceName from gpu_util.go as key?
```

Comment:
If we have this constant defined somewhere in the codebase, we should probably use it. 

Can you either remove this TODO, or rephrase as suggested above?

Reply:
It is defined. But it is defined in e2e/framework module. Which does not seem like a nice dependency of test/runners module. Hence the question.

I will of course drop the TODO before it is merged. I put it just for sake of review so we settle out the correct code shape.